### ISSUE TITLE
Bug: Crash on opening a specific profile

### ISSUE THREAD

**Issue by Murgeye** [Dimillian/IceCubesApp#1949]
## Environment:

 - OS: iOS 17.3
 - IceCubesApp version: 1.10.30

## Description

Opening the profile https://shitposter.club/users/condret and scrolling down a bit crashes the app. Toots by that person in a thread are invisible as well: https://infosec.exchange/@pancake/111884038345144356

## Related Issues

 - [x] Search that this bugs don't already exist before creating

Found none open right now.

**Comment by Dimillian** [comment]
did you blocked him?

**Comment by Murgeye** [comment]
No, the account is neither blocked nor muted. I only clicked on the profile because the comments were invisible in the thread.

**Comment by Murgeye** [comment]
Ah, but it seems to have been "hidden" by my instance - might be that he is blocked on my instance? (Other profiles from his instance work fine).
